docs/source/modules/tutorial.rst

```diff
@@ -0,0 +1,653 @@
+   for those that have already submitted a model like AlexNet
+   and have a submission package ready. If you have not done this,
+   I highly recommend going through this tutorial beforehand, or else you will
+   encounter some errors along the way.
+2. The entire package we submit will be the same as a pretrained model,
+   but with the ``models/base_models.py`` file different (as the model itself is different).
+   So, I would recommend just copying the ``my_alexnet_submission`` folder,
+   pasting it into the ``my_model_submissions`` folder, and renaming it to something
+   like ``my_custom_submission``. This will take care of all the tricky
+   submission stuff, and you can just focus on implementing the actual model inside ``models/base_models.py``.
+3. Now the fun part: scoring a model that you create! In this section we will be implementing
+   a light-weight Pytorch model and submitting that. All this entails is adding
+   a little bit of extra stuff to ``models/base_models.py``.
+4. The easiest way to do this is to simply copy all the code in the block below,
+   and I can walk you through the important stuff that is necessary
+   to understand how to submit a custom model. It is, in a nutshell, just a
+   slightly more complicated version of the original ``base_models.py`` template
+   in the ``sample-model-submissions`` folder. The code is listed below ::
+
+    # Custom Pytorch model from:
+    # https://github.com/brain-score/candidate_models/blob/master/examples/score-model.ipynb
+
+    from model_tools.check_submission import check_models
+    import numpy as np
+    import torch
+    from torch import nn
+    import functools
+    from model_tools.activations.pytorch import PytorchWrapper
+    from brainscore import score_model
+    from model_tools.brain_transformation import ModelCommitment
+    from model_tools.activations.pytorch import load_preprocess_images
+    from brainscore import score_model
+
+    """
+    Template module for a base model submission to brain-score
+    """
+
+    # define your custom model here:
+    class MyModel(nn.Module):
+        def __init__(self):
+            super(MyModel, self).__init__()
+            self.conv1 = torch.nn.Conv2d(in_channels=3, out_channels=2, kernel_size=3)
+            self.relu1 = torch.nn.ReLU()
+            linear_input_size = np.power((224 - 3 + 2 * 0) / 1 + 1, 2) * 2
+            self.linear = torch.nn.Linear(int(linear_input_size), 1000)
+            self.relu2 = torch.nn.ReLU()  # can't get named ReLU output otherwise
+
+        def forward(self, x):
+            x = self.conv1(x)
+            x = self.relu1(x)
+            x = x.view(x.size(0), -1)
+            x = self.linear(x)
+            x = self.relu2(x)
+            return x
+
+
+    # init the model and the preprocessing:
+    preprocessing = functools.partial(load_preprocess_images, image_size=224)
+
+    # convert the ANN in this case to a brain-model object to test neural benchmarks:
```

Comment:
no, the language here is off. This will give us an activations model. BrainModel is what comes out of wrapping the activations model with `ModelCommitment` (see comments in https://github.com/brain-score/candidate_models/blob/master/examples/score-model.ipynb)